packages/allure-cucumberjs/src/CucumberJSAllureReporter.ts

```diff
@@ -299,13 +299,18 @@ export class CucumberJSAllureFormatter extends Formatter {
     const currentTest = new AllureTest(this.allureRuntime, Date.now());
     const thread = data.workerId || ALLURE_THREAD_NAME || process.pid.toString();
     const fullName = `${pickle.uri}#${pickle.name}`;
+    // sometimes description starts with "Description:", but it's not actually required
+    const description = scenario?.description
+      ? scenario.description.replace(/\s+Description:\s?/, "")
+      : doc?.feature?.description;
```

Comment:
Maybe always join the feature description with the scenario description?

Reply:
I afraid it'll produce strange descriptions with a lot of duplicated data. Guess, we should entirely replace feature description by a local one